### PR TITLE
docs: specs added

### LINKED ISSUE #2200: Docs: add system requirements in Quick Start
## Feature Request
Add minimal system requirements to run Talos
### Description
There should be a list of system requirements to run Talos.
For instance: 

- 4GB ram
- 2 CPU
- 10GB Disk

### PR DESCRIPTION
**Pull request by patatman**
specs added to the quickstart, to fix #2200

Signed-off-by: Patatman <git@jeursen.nl>

# Pull Request

## Acceptance

Please use the following checklist:

- [ ] you linked an issue (if applicable)
- [ ] you included tests (if applicable)
- [x] you ran conformance (`make conformance`)
- [ ] you formatted your code (`make fmt`)
- [x] you linted your code (`make lint`)
- [ ] you generated documentation (`make docs`)
- [ ] you ran unit-tests (`make unit-tests`)

> See `make help` for a description of the available targets.

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/2205)

### CODE REVIEW

File: docs/website/content/v0.6/en/guides/getting-started/intro.md [hunk 1]
```diff
@@ -19,3 +19,57 @@ There are two options:
 
 - [Run a Docker-based local cluster](/docs/v0.5/en/guides/local/docker) on your Linux or Mac workstation
 - [Run a Firecracker micro-VM-based](/docs/v0.5/en/guides/local/firecracker) cluster on your Linux workstation
+
+### System requirements
+
+Talos itself is a low resource OS, but since Talos goal is to run Kubernetes you need to have atleast the following hardware requirements:
```
Comment (rsmitty) [inline docs/website/content/v0.6/en/guides/getting-started/intro.md:25]: Super tiny typo, this needs to be `at least`

File: docs/website/content/v0.6/en/guides/getting-started/intro.md [hunk 2]
```diff
@@ -19,3 +19,57 @@ There are two options:
...
+    </tr>
+    <tr class="bg-gray-100">
+      <td class="border px-4 py-2">Worker</td>
+      <td class="border px-4 py-2">2GB</td>
+      <td class="border px-4 py-2">2</td>
+    </tr>
+  </tbody>
+</table>
+
+These requirements are similair to that of kubernetes.
```
Comment (rsmitty) [inline docs/website/content/v0.6/en/guides/getting-started/intro.md:75]: "similar"

**Comment by rsmitty** [comment]
If we can get the nits fixed and rebased, we'll merge ASAP.

**Comment by patatman** [comment]
@rsmitty Fixed the typo's 🙂

**Pull request merged**